### PR TITLE
Change random bits and pieces, again

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

Why do I keep finding stuff like this ..

# Result

Not really any visual changes worth showing.

# Testing

- [x] I have thoroughly tested my changes.

No breaking changes.

### CODE REVIEW

File: app/routes/admin/email/components/EmailUsers.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default class EmailUsers extends Component<Props> {
         ),
       },
       {
-        title: 'Internepost',
+        title: 'Interne-post',
```
Comment (eikhr) [inline app/routes/admin/email/components/EmailUsers.tsx:40]: This looks a bit weird, I think "Intern e-post" would be better

File: app/routes/admin/email/components/RestrictedMailEditor.tsx [hunk 2]
```diff
@@ -155,7 +155,7 @@ const RestrictedMailEditor = ({
 
       {!restrictedMailId && (
         <Button submit disabled={submitting}>
-          Lag flaskepost
+          Lag flaske-post
```
Comment (eikhr) [inline app/routes/admin/email/components/RestrictedMailEditor.tsx:158]:
```suggestion
          Lag flaskepost
```
Unintentional?

File: app/routes/pages/components/LandingPage.tsx [hunk 3]
```diff
@@ -86,7 +86,7 @@ const LandingPage: PageRenderer = ({ loggedIn }) => {
             email="abakus@abakus.no"
             logo="https://raw.githubusercontent.com/webkom/lego/master/assets/abakus_hs.png"
           />
-          <h3 className={styles.title}>E-postadresser til komiteene</h3>
+          <h3 className={styles.title}>E-post-adresser til komiteene</h3>
```
Comment (eikhr) [inline app/routes/pages/components/LandingPage.tsx:89]:
```suggestion
          <h3 className={styles.title}>E-postadresser til komiteene</h3>
```
No hyphen here https://ordbokene.no/bm/70008

File: app/routes/users/components/StudentConfirmation.tsx [hunk 4]
```diff
@@ -80,13 +80,13 @@ const StudentConfirmation = ({
   return (
     <Container>
       <div>
-        <h2>Verifiser studentepost</h2>
+        <h2>Verifiser studente-post</h2>
```
Comment (eikhr) [inline app/routes/users/components/StudentConfirmation.tsx:83]:
```suggestion
        <h2>Verifiser student e-post</h2>
```
Imo. this looks better with a space, but I'm happy as long as we are consistent about it

**Review by eikhr — APPROVED** [review]
Great work, fixing small things like this really contributes to making the page feel more professional

File: app/routes/admin/email/components/RestrictedMailEditor.tsx (hunk 2 above)
Comment (ivarnakken) [reply app/routes/admin/email/components/RestrictedMailEditor.tsx:158]: Haha yup

File: app/routes/users/components/StudentConfirmation.tsx (hunk 4 above)
Comment (ivarnakken) [reply app/routes/users/components/StudentConfirmation.tsx]: I've now found examples of OsloMet, Kristiania and UiB writing "student-e-post" with a hyphen, so I'll go with that. Nice catch

**Review by ollfkaih — APPROVED** [review]

**Pull request merged**